static/js/recent_topics.js

```diff
@@ -528,13 +548,10 @@ export function show() {
     $("#message_view_header_underpadding").hide();
     $(".header").css("padding-bottom", "0px");
 
-    // Save text in compose box if open.
-    drafts.update_draft();
-    // Close the compose box, this removes
-    // any arbitrary bug for compose box in recent topics.
-    // This is required since, Recent Topics is the only view
-    // with no compose box.
-    compose_actions.cancel();
```

Comment:
Why the change to drafts here?  Oh, I guess because we're no longer closing the compose box, if it were open?